[2021-09-28 15:27:19,041] {processor.py:162} INFO - Started process (PID=108) to work on /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:27:19,062] {processor.py:613} INFO - Processing file /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py for tasks to queue
[2021-09-28 15:27:19,283] {logging_mixin.py:109} INFO - [2021-09-28 15:27:19,277] {dagbag.py:496} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:27:54,633] {logging_mixin.py:109} INFO - [2021-09-28 15:27:54,590] {timeout.py:36} ERROR - Process timed out, PID: 108
[2021-09-28 15:27:54,660] {logging_mixin.py:109} WARNING - Exception ignored in: <function WeakValueDictionary.__init__.<locals>.remove at 0x7f7ac2872ea0>
[2021-09-28 15:27:54,664] {logging_mixin.py:109} WARNING - Traceback (most recent call last):
[2021-09-28 15:27:54,672] {logging_mixin.py:109} WARNING -   File "/usr/local/lib/python3.6/weakref.py", line 109, in remove
[2021-09-28 15:27:54,688] {logging_mixin.py:109} WARNING -     def remove(wr, selfref=ref(self), _atomic_removal=_remove_dead_weakref):
[2021-09-28 15:27:54,712] {logging_mixin.py:109} WARNING -   File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/timeout.py", line 37, in handle_timeout
[2021-09-28 15:27:54,850] {logging_mixin.py:109} WARNING -     raise AirflowTaskTimeout(self.error_message)
[2021-09-28 15:27:54,906] {logging_mixin.py:109} WARNING - airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py after 30.0s, PID: 108
[2021-09-28 15:28:00,563] {processor.py:623} INFO - DAG(s) dict_keys(['example_trigger_target_dag']) retrieved from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:51:36,196] {processor.py:162} INFO - Started process (PID=53) to work on /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:51:36,218] {processor.py:613} INFO - Processing file /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py for tasks to queue
[2021-09-28 15:51:36,227] {logging_mixin.py:109} INFO - [2021-09-28 15:51:36,227] {dagbag.py:496} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:51:36,476] {processor.py:623} INFO - DAG(s) dict_keys(['example_trigger_target_dag']) retrieved from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:52:38,759] {processor.py:162} INFO - Started process (PID=86) to work on /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:52:38,864] {processor.py:613} INFO - Processing file /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py for tasks to queue
[2021-09-28 15:52:39,181] {logging_mixin.py:109} INFO - [2021-09-28 15:52:39,049] {dagbag.py:496} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:52:40,920] {processor.py:623} INFO - DAG(s) dict_keys(['example_trigger_target_dag']) retrieved from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:53:54,688] {processor.py:162} INFO - Started process (PID=126) to work on /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:53:54,980] {processor.py:613} INFO - Processing file /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py for tasks to queue
[2021-09-28 15:53:55,207] {logging_mixin.py:109} INFO - [2021-09-28 15:53:55,181] {dagbag.py:496} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:53:59,847] {processor.py:623} INFO - DAG(s) dict_keys(['example_trigger_target_dag']) retrieved from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:55:48,661] {processor.py:162} INFO - Started process (PID=184) to work on /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:55:49,147] {processor.py:613} INFO - Processing file /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py for tasks to queue
[2021-09-28 15:55:49,524] {logging_mixin.py:109} INFO - [2021-09-28 15:55:49,524] {dagbag.py:496} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:57:53,216] {processor.py:162} INFO - Started process (PID=249) to work on /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:57:53,681] {processor.py:613} INFO - Processing file /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py for tasks to queue
[2021-09-28 15:57:53,775] {logging_mixin.py:109} INFO - [2021-09-28 15:57:53,760] {dagbag.py:496} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 15:58:32,485] {logging_mixin.py:109} INFO - [2021-09-28 15:58:25,759] {timeout.py:36} ERROR - Process timed out, PID: 249
[2021-09-28 15:58:34,468] {logging_mixin.py:109} INFO - [2021-09-28 15:58:32,936] {dagbag.py:329} ERROR - Failed to import: /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 326, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py", line 27, in <module>
    from airflow.operators.python import PythonOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/python.py", line 30, in <module>
    from airflow.exceptions import AirflowException
  File "<frozen importlib._bootstrap>", line 1007, in _handle_fromlist
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/timeout.py", line 37, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py after 30.0s, PID: 249
[2021-09-28 15:58:35,853] {processor.py:625} WARNING - No viable dags retrieved from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 16:00:15,387] {processor.py:162} INFO - Started process (PID=295) to work on /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 16:00:15,708] {processor.py:613} INFO - Processing file /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py for tasks to queue
[2021-09-28 16:00:15,866] {logging_mixin.py:109} INFO - [2021-09-28 16:00:15,811] {dagbag.py:496} INFO - Filling up the DagBag from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
[2021-09-28 16:00:22,023] {processor.py:623} INFO - DAG(s) dict_keys(['example_trigger_target_dag']) retrieved from /home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_trigger_target_dag.py
